src/sidebar/store/modules/selection.js

```diff
@@ -318,7 +318,7 @@ function setForceVisible(id, visible) {
 function focusAnnotations(tags) {
   return {
     type: actions.FOCUS_ANNOTATIONS,
-    focused: freeze(toSet(tags)),
+    focusedTags: tags,
```

Comment:
I don't believe this was necessary, but please alert me if I'm being naive.

Reply:
We now freeze all top-level state in dev builds, so indeed this is obsolete.